helm-charts/splunk-otel-collector/templates/secret-validation-hook.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if not .Values.secret.create }}
+# Helm hook validating that custom secret provided by user has all the required
+# fields.
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ template "splunk-otel-collector.fullname" . }}-validate-secret
+  labels:
+    {{- include "splunk-otel-collector.commonLabels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-upgrade,pre-install
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+spec:
+  restartPolicy: Never
+  containers:
+  - name: validate-secret
+    image: {{ template "splunk-otel-collector.image.otelcol" . }}
+    imagePullPolicy: {{ .Values.image.otelcol.pullPolicy }}
+    command: ["sh", "-c"]
```

Comment:
this image doesn't have `sh`, no?

Reply:
It does have it

Reply:
oh i see it does. but how? just curious.

Reply:
It's being picked up from signalfx-agent bundle https://github.com/signalfx/signalfx-agent/blob/main/Dockerfile#L306-L332